less/pages/clubs-form.less

```diff
@@ -4,11 +4,20 @@
     width: 80%;
     margin: auto;
 
+    * {
+      // serious, get out of here, outline.
+      outline: 0 !important;
+    }
```

Comment:
Hmm. Not so sure about this?

If we are gonna add this rule we shouldn't put it in this file, but in one of the more "globally oriented" ones.